src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -552,12 +549,14 @@ def dbs_get_all(dbs, db_name, _hash, *args, **kwargs):
         db get_all function executed on global and all namespace DBs.
         """
         result = {}
+        # If there are multiple namespaces, _hash might not be 
+        # present in all namespace, ignore if not present in a
+        # specfic namespace.
+        if (len(dbs) > 1) : kwargs['blocking'] = False
```

Comment:
>kwargs [](start = 28, length = 6)

This parameter is passed as reference, and it is mutable.
If you change it inside function, it will have side-effect to the caller's scope. Could you do not change it?

Reply:
@qiluo-msft We were initially updating all the function call arguments and the changes were spread in multiple places. This approach looks better where only for Multi-Asic we are making blocking as False and for single asic behavior remains unchanged.
Also caller should not be impacted by this as it does not get used in caller scope and just passes the value to the library.

Reply:
As per suggestion, instead of modifying kwargs directly, did a shallow copy to change.